test/JDBC/upgrade/latest/schedule

```diff
@@ -215,6 +215,7 @@ forjson
 forjson-datatypes
 forjson-subquery
 forjson-nesting
+forjsonauto
```

Comment:
If we already support for json auto syntax, then we should add this test case into other Mvu schedule files as well.